source/connect/connect-to-mongodb.txt

```diff
@@ -0,0 +1,145 @@
+.. _pymongo-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide shows you how to use {+driver-short+} to connect to a MongoDB instance or replica set
+deployment.
+
+.. _pymongo_connection_uri:
+
+Connection URI
+--------------
+
+A **connection URI**, also known as a *connection string*, tells the driver how to
+connect to a MongoDB deployment and how to behave while connected.
+
+A standard connection string includes the following pieces:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Piece
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password@``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``. 
```

Comment:
> authSource

What is `authSource` referring to here? I don't see any references to it besides the two in this breakdown block.

Reply:
added a link to the auth page.